cms/utils/compat/forms.py

```diff
@@ -1,6 +1,11 @@
 # -*- coding: utf-8 -*-
```

Comment:
This line is probably unnecessary in this very source file, and could be removed.